effect/src/test/scala/org/typelevel/catbird/util/effect/RerunnableContextShiftSuite.scala

```diff
@@ -42,6 +42,7 @@ class RerunnableContextShiftSuite extends FixtureAnyFunSuite with ThreadPoolNami
     assert(poolName1 == f.futurePoolName)
     assert(poolName2 == f.ioPoolName)
     assert(poolName2 == f.ioPoolName)
+    assert(poolName3 == f.futurePoolName)
```

Comment:
`poolName3` was previously unused. I'm not 100% sure if this is the right test, but it passes.